### PR TITLE
transferred navbar changes from impact-page-tanaya

### PR DESCRIPTION
**Pull request by tnawathe21**
Very rough change -- added initiatives section to the mobile and desktop view of the website

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -7,21 +7,29 @@ import React from 'react';
...
-				<li><Link href={'/committees#studio'}><a>Studio</a></Link></li>
-				<li><Link href={'/committees#icpc'}><a>ICPC</a></Link></li>
-				<li><Link href={'/committees#design'}><a>Design</a></Link></li>
-				<li><Link href={'/committees#cyber'}><a>Cyber</a></Link></li>
-				<li><Link href={'/committees#teachla'}><a>Teach LA</a></Link></li>
-				<li><Link href={'/committees#w'}><a>W</a></Link></li>
-				<li><Link href={'/committees#ai'}><a>AI</a></Link></li>
-				<li><Link href={'/committees#hack'}><a>Hack</a></Link></li>
+					<li><Link href={'/impact'}><a>Impact</a></Link></li>
+					<li><Link href={''}><a>Jedi</a></Link></li>
```
Comment (mattxwang) [inline components/Navbar.js:29]:
```suggestion
					<li><Link href={'/jedi'}><a>JEDI</a></Link></li>
```

File: components/Navbar.js [hunk 2]
```diff
@@ -67,20 +75,19 @@ export default class Navbar extends React.Component {
...
 						</a>
 					</Link>
-					<div className="nav-section right" id="desktop-nav" role="navigation">
+					<div className="nav-section right" id="desktop-nav">
 						<ul className="nav-items">
 							<li><Link href="/about"><a>About</a></Link></li>
 							<li><Link href="/committees"><a>Committees</a></Link></li>
 							{/* <li><Link href="/gm"><a>{gmData.date.quarter} GM</a></Link></li> */}
 							<li><Link href="/events"><a>Events</a></Link></li>
+							<li className="initiatives"><Link href="/impact"><a>Initiatives</a></Link></li>
```
Comment (mattxwang) [inline components/Navbar.js:84]:
```suggestion
```

For now, let's remove this - we'll later make an initiatives page.

**Review by mattxwang — CHANGES_REQUESTED** [review]
Thanks Tanaya! A couple of quick things:

* do you mind resolving the merge conflict? happy to lend a hand if it's a struggle
* can you take a look at the next build error?

Also left some tiny code comments below!

**Comment by mattxwang** [comment]
@advaithg tagging you in on this

**Comment by mattxwang** [comment]
@advaithg do you plan on taking this up? If not, going to toss it to @reginawang3495 and TLA

**Comment by advaithg** [comment]
> going to toss it to @reginawang3495 and TLA

sounds good! I'm fine w that!

**Comment by mattxwang** [comment]
See #351.

**Pull request closed without merging**